database/gormdb_assignment.go

```diff
@@ -156,8 +152,7 @@ func (db *GormDB) CreateCriterion(query *qf.GradingCriterion) error {
 // UpdateCriterion updates the given criterion
 func (db *GormDB) UpdateCriterion(query *qf.GradingCriterion) error {
 	return db.conn.
-		Where(&qf.GradingCriterion{ID: query.ID, BenchmarkID: query.BenchmarkID}).
-		Updates(&qf.GradingCriterion{Description: query.Description, Comment: query.Comment, Grade: query.Grade, Points: query.Points}).Error
+		Where(&qf.GradingCriterion{ID: query.ID, BenchmarkID: query.BenchmarkID}).Select("*").Updates(query).Error
```

Comment:
Long line. Please break up the where clause similar to the `UpdateBenchmark` above.

Also, I think select should be before the where clause.

Reply:
```go
	return db.conn.
		Select("*").
		Where(&qf.GradingCriterion{
			ID:          query.ID,
			BenchmarkID: query.BenchmarkID,
		}).Updates(query).Error
```